pvactools/lib/aggregate_all_epitopes.py

```diff
@@ -452,6 +455,17 @@ def get_tier(self, mutation, vaf_clonal):
             if mutation["Transcript Support Level"] > self.maximum_transcript_support_level:
                 tsl_pass = False
 
+        ic50_pass = mutation["{} MT IC50 Score".format(self.mt_top_score_metric)] < binding_threshold
+        percentile_pass = (
+            self.percentile_threshold is not None and 
+            mutation["{} MT Percentile".format(self.mt_top_score_metric)] < self.percentile_threshold
+        )
+        thresholds_passed = (
+            (ic50_pass and percentile_pass) 
+            if self.percentile_threshold_strategy == 'conservative' 
+            else (ic50_pass or percentile_pass)
+        )
+
```

Comment:
Might be getting too nitpicky here, but I think I would prefer to have this block of code be right after setting the `binding_threshold` (line 445) since they are logically related. I think it would make for easier readability.